src/main/java/com/conveyal/datatools/editor/controllers/api/EditorController.java

```diff
@@ -204,7 +205,11 @@ private String patchTable(Request req, Response res) {
             }
             for (int i = 0; i < filterFields.size(); i++) {
                 Field field = filterFields.get(i);
-                field.setParameter(preparedStatement, oneBasedIndex, filterValues.get(i));
+                try {
+                    field.setParameter(preparedStatement, oneBasedIndex, filterValues.get(i));
+                } catch (Exception e) {
```

Comment:
Can this Exception type be more specific?

Reply:
I'd prefer to keep it general because if an exception that we don't anticipate is thrown, no error feedback will be returned to the user at all.